### DISCUSSION TITLE
Android Fullscreen has a black bar covering the camera notch

### DISCUSSION THREAD

**Discussion by MasterXYZ5** [ppy/osu#17769] [Q&A] [answered]
![Screenshot_20220411-143343~2](https://user-images.githubusercontent.com/69831140/162740911-63c94912-9aac-4025-be42-ca1778175e0a.png)
The Android version of OSU! Lazer doesn't let me set the game to immersive fullscreen, aka fullscreen over the camera notch. I've tried playing around with the screen scaling option and also forcing it from my device settings, but with no results. Is it supposed to be like that? Shouldn't there be an option to let users choose if they want the game to cover the notch or go fullscreen over it? Thanks for any response in advance

**Comment by integerrr** [comment]
Please at least post a picture of what you're getting and how you expect the game to look in full screen, I have no idea what you mean at the moment

**Reply by MasterXYZ5** [reply]
Updated the post to include the screenshot

**Comment by peppy** [comment, marked as answer]
This is intentional (implemented at our end). One day we may offer a better solution, but it the game was to draw in that area you would not be able to press buttons.

**Reply by MasterXYZ5** [reply]
Ok thanks for the answer, I wasn't sure if this was intentional or not considering that many other games and apps just extend the UI over the notch area but keep the buttons away from it

**Reply by peppy** [reply]
That is the eventual goal.
